what's new.md

```diff
@@ -284,6 +284,7 @@
 
 - Faster scalar's WithinEpsilon with Math.abs ([nekochanoide](https://github.com/nekochanoide))
 - Added `decomposeToTransformNode` ([RaananW](https://github.com/RaananW))
+- Normalized types and return values in math.vector.ts ([LostInClams](https://github.com/LostInClams))
```

Comment:
please revert this, there is no need to change the what's new file :-)

Reply:
Should I fix line 63 in contributing.md as well?

"- Make sure to add a line about your PR in the [what's new](https://github.com/BabylonJS/Babylon.js/blob/master/what's%20new.md)"

Reply:
Yep! thanks for that :-)